packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -455,6 +482,17 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
               })`}
               className={this._classNames.xAxis}
             />
+            <SVGTooltipText
```

Comment:
enable this SVGTooltipText element only if axis title is defined. 
That would mean no existing test snapshot will change

Reply:
done